airflow/ui/vite.config.ts

```diff
@@ -30,7 +30,7 @@ export default defineConfig({
     {
       name: "transform-url-src",
       transformIndexHtml: (html) =>
-        html.replace(`src="./assets/`, `src="./static/assets/`).replace(`href="/`, `href="/webapp/`),
+        html.replace(`src="./assets/`, `src="./static/assets/`).replace(`href="/`, `href="./`),
```

Comment:
Question: What path does the browser request for when you click a link directly to a sub-page? i.e. if you navigate to a TI page etc, and then open a new tab directly at that URL, does the browser see the base as `/webapp/` or the current url and try to request `/webapp/dag/<dag>/.../static/pin_32.png`? Maybe not for that file, but other images. that's my concern.

Reply:
You're right, that's not handled properly at the moment.

Let me take a look.

Reply:
`<base href="">` might be a fix for that?

Reply:
Yes base might be good.

Reply:
https://github.com/apache/airflow/pull/46933/commits/9adf8628a1086438cf2c7848d3525299dc793092

Reply:
Base is working nicely